lisa/tools/gcc.py

```diff
@@ -50,5 +50,26 @@ def get_version(self) -> VersionInfo:
 
     def _install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
-        posix_os.install_packages("gcc")
+        if isinstance(posix_os, BSD):
```

Comment:
Do you need to proceed BSD now, or leaving it later?

Reply:
going to remove it since gcc build of sockperf doesn't work, a c/c++ compiler clang is shipped by default for freebsd at least so that code isn't needed here.